components/ProfileTasks.js

```diff
@@ -21,8 +21,10 @@ export default function ProfileTasks(props) {
                   className="flex items-center underline text-deep-blue-dark hover:text-blue-hover "
                   data-cy="task-link"
                   onClick={(e) => {
-                    e.preventDefault()
-                    props.openModal(task.link)
+                    if (task.betaPopUp) {
```

Comment:
Don't forget to update the task shape to include betaPopUp at the bottom of the file

ProfileTasks.propTypes = {
  dataCy: PropTypes.string,
  openModal: PropTypes.func,
  tasks: PropTypes.arrayOf(
    PropTypes.shape({
      title: PropTypes.string.isRequired,
      areaLabel: PropTypes.string.isRequired,
      link: PropTypes.string.isRequired,
      icon: PropTypes.string.isRequired,
    })
  ),